diff --git a/node_modules/react-native-worklets/lib/module/threads.js b/node_modules/react-native-worklets/lib/module/threads.js
index 9a337f4..3648b58 100644
--- a/node_modules/react-native-worklets/lib/module/threads.js
+++ b/node_modules/react-native-worklets/lib/module/threads.js
@@ -297,7 +297,7 @@ export function runOnUIAsync(worklet) {
     });
   };
 }
-if (__DEV__) {
+if (__DEV__ && !SHOULD_BE_USE_WEB) {
   function runOnUIAsyncWorklet() {
     'worklet';
 
